exporters/elasticsearch/src/es_log_exporter.cc

```diff
@@ -74,23 +77,19 @@ class ResponseHandler : public http_client::EventHandler
     switch (state)
     {
       case http_client::SessionState::ConnectFailed:
-        if (console_debug_)
-          std::cout << "Connection to elasticsearch failed\n";
+        OTEL_ERROR("[ES Trace Exporter] Connection to elasticsearch failed\n")
```

Comment:
I would've used something like `OTEL_LOG_ERROR` + reserved the entire set of corresponding macros for warnings, notices, etc.

Reply:
Good point. Will change that.